### PR TITLE
PHPUnit v6 support

### PR DESCRIPTION
**Pull request by MarioBlazek**
This PR enables PHPUnit v6 support for this repo.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -39,8 +39,8 @@
     "require-dev": {
         "friendsofphp/php-cs-fixer": "~2.7.1",
         "mikey179/vfsStream": "1.1.0",
-        "phpunit/phpunit": "^5.7",
-        "matthiasnoback/symfony-dependency-injection-test": "~1.0",
+        "phpunit/phpunit": "~6.0",
```
Comment (emodric) [inline composer.json]: Unless targeting 7.0, this should remain at `^5.7|^6.0`, due to 6.0 not supporting PHP 5.6.

File: composer.json [hunk 2]
```diff
@@ -39,8 +39,8 @@
...
+        "matthiasnoback/symfony-dependency-injection-test": "~2.0",
```
Comment (emodric) [inline composer.json]: Like the above comment, this should be `^1.0|^2.0` to support testing on both PHP 5.6 and PHP 7.x

Comment (andrerom) [reply composer.json]: master does not support php 5.6 anymore ;)

Comment (emodric) [reply composer.json]: 👍

**Comment by MarioBlazek** [comment]
ping @andrerom @emodric please review

File: composer.json (hunk 1 above)
Comment (andrerom) [reply composer.json]: @MarioBlazek Like in LegacyBridge, could we use tilde here just for clarity?

File: bootstrap.php [hunk 3]
```diff
@@ -20,4 +20,6 @@
     throw new \RuntimeException('Could not read config.php, please copy config.php-DEVELOPMENT to config.php & customize to your needs!');
 }
 
+class_alias('PHPUnit\Framework\TestCase', 'PHPUnit_Framework_TestCase');
```
Comment (andrerom) [inline bootstrap.php]: Maybe add comment that this is for BC for code written using old form?

Comment (emodric) [reply bootstrap.php]: Is this even needed? What dependencies are using non namespaced variant of this class?

Comment (alongosz) [reply bootstrap.php]: > What dependencies are using non namespaced variant of this class?

Kernel tests are executed from meta-repo also, so we need to check if that works anyway.

**Comment by MarioBlazek** [comment]
@andrerom PR updated as requested

**Pull request merged**